src/NetMQ.Tests/ClientServer.cs

```diff
@@ -82,10 +82,9 @@ public async Task AsyncWithCancellationToken()
             await Assert.ThrowsAnyAsync<OperationCanceledException>(async () => await server.ReceiveStringAsync(source.Token));
         }
 
-#if NETCOREAPP3_1
-
+#if NETSTANDARD || NET
```

Comment:
I still get tripped up by pragma defines but this pragma only enabled the test on net5+ (dotnet test 4.7.2 does not pick it up according to the CI logs)

I imagine this pragma was part of when certain targets didn't support IAsyncEnumerable? With netstandard2.0, net4.7.2 and net8+ supporting it, I imagine this pragma can removed let the tests run always?

Reply:
@chrbauer want to file that as a PR? Seems like a good change.